packages/rollup-plugin-minify-html-literals/src/index.ts

```diff
@@ -0,0 +1,79 @@
+import {
```

Comment:
This also applies to every other .ts sourcefile here, but I'm unsure what the correct thing to do here with regards to license headers are. We might need to add Google license header to every file, along side attribution to Liz too?

Reply:
Hi @augustjk , thanks for the initial review with the comments and feedback. I have followed your suggestions and pushed the changes. 

For this one, I recall speaking about this in one of the engineering meetings. I was not sure if we needed it for every file. In the LICENSE.md I do have both in there. If needed, I will go through and add the license header to all the ts files.

Reply:
We're good for now. Let's not add the license header during import since the code as they are retain the original copyright. Once we start editing any files, we'll need to add the Google license header then, but that'll be for later.